domain/article.go

```diff
@@ -0,0 +1,36 @@
+package domain
```

Comment:
Ask: Why not ```domains``` (plural)?

Reply:
Could you please NOT message in my comments and message directly to the author

Reply:
Comments here make good read for new people like us, to understand why this way and not the other way. also pull requests are for code review and comments right?

Reply:
I agree with that, but thats not my point.
If you scroll down to the end of this page, there is a comment box, what he has done is replied to my comment similar to what you are doing

Reply:
Ohh I see missed that in a long page scroll.